### ISSUE TITLE
Broken Build with @cycle/dom 15.0.0-rc.1

### ISSUE THREAD

**Issue by jonsharratt** [cyclejs-community/create-cycle-app#65]
**Code to reproduce the issue:**
1. Clean creation as of 21st Jan 2017
2. `create-cycle-app my-cycle-app`
    `ES6 (babel) + Webpack`
    `XStream, tailored for Cycle.js`
3. `cd my-cycle-app`
4. `npm start`

**Expected behavior:**
```
Build completed in [X]s

App is running at http://localhost:8000

webpack: bundle is now VALID.
```

**Actual behavior:**
```
ERROR in ./~/@cycle/dom/lib/MainDOMSource.js
Module not found: Error: Cannot resolve module '@cycle/run/lib/adapt' in /Users/jon/Documents/git/craftship/tech-task-cycle/node_modules/@cycle/dom/lib
 @ ./~/@cycle/dom/lib/MainDOMSource.js 2:14-45

ERROR in ./~/@cycle/dom/lib/HTMLSource.js
Module not found: Error: Cannot resolve module '@cycle/run/lib/adapt' in /Users/jon/Documents/git/craftship/tech-task-cycle/node_modules/@cycle/dom/lib
 @ ./~/@cycle/dom/lib/HTMLSource.js 3:14-45

ERROR in ./~/@cycle/dom/lib/mockDOMSource.js
Module not found: Error: Cannot resolve module '@cycle/run/lib/adapt' in /Users/jon/Documents/git/craftship/tech-task-cycle/node_modules/@cycle/dom/lib
 @ ./~/@cycle/dom/lib/mockDOMSource.js 3:14-45

ERROR in ./~/@cycle/dom/lib/DocumentDOMSource.js
Module not found: Error: Cannot resolve module '@cycle/run/lib/adapt' in /Users/jon/Documents/git/craftship/tech-task-cycle/node_modules/@cycle/dom/lib
 @ ./~/@cycle/dom/lib/DocumentDOMSource.js 3:14-45

ERROR in ./~/@cycle/dom/lib/BodyDOMSource.js
Module not found: Error: Cannot resolve module '@cycle/run/lib/adapt' in /Users/jon/Documents/git/craftship/tech-task-cycle/node_modules/@cycle/dom/lib
 @ ./~/@cycle/dom/lib/BodyDOMSource.js 3:14-45
```

**Versions of packages used:**
```
  "dependencies": {
    "@cycle/dom": "^15.0.0-rc.1",
    "@cycle/xstream-run": "^4.2.0",
    "xstream": "^9.3.0"
  },
```

Looks like changes are afoot (https://github.com/cyclejs/cyclejs/commit/b8a7c3649a5ea8d5c4eed826b77d949575b244ca) as the import referred to is not on master yet, if anyone else hits this issue just now I used:

1. `npm run take-off-training-wheels`
2. Edit `package.json` to use `"@cycle/dom": "14.3.0"`
3. `npm i`
4. `npm start`

Think I literally have just caught this stuff as `v15-rc.1` is in flux, although does ask the question whether the `create-cycle-app` should lock down major versions explicitly when you `init` the project and let only major and minors through to stop this happening going forward.

Additionally could setup nightly builds in CI of the project with relevant tests that ensure the app still integrates correctly.

Thoughts?

**Comment by nickbalestra** [comment]
Thanks for reporting this @jonsharratt!
Yes, I think we should lock down major. I also agree on CI. I'm working on v3 branch where some of this issues should be addressed. I'll fix this and release an update.

**Comment by nickbalestra** [comment]
@jonsharratt if you want feel free to give the next version a spin:

```
npm i -g create-cycle-app@next
```

Is a WIP, feel free to follow dev and jump in with any idea/feedback on the relative issue/branch/pr

**Comment by jonsharratt** [comment]
Hey Nick, cheers for this! will take a look this week and start raising some issues for discussion before implementation.  Closing this issue as all in hand with the future improvements 👯

**Comment by nickbalestra** [comment]
I'll rather leave this open for others until fixed.

**Comment by nickbalestra** [comment]
Fixed with v3 release #63
